src/test/data/VersionControlTest/VersionControlTest/Commit/24571d697e6708860610f27e36d96a12a6872d9d

```diff
@@ -0,0 +1,48 @@
+{
+  "students" : [ {
+    "name" : "Bernice Yu",
+    "phone" : "NA",
+    "email" : "berniceyu@example.com",
+    "telegram" : "@berniceyu",
+    "attendance" : [ false, false, false, false, false, false, false, false, false, false, false, false ],
+    "participation" : [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ],
+    "assessment" : {
+      "RA2" : 4,
+      "FINAL" : 50,
+      "RA1" : 34,
+      "MIDTERM" : 13,
+      "PE" : 59
+    },
+    "tagged" : [ "colleagues", "friends" ]
```

Comment:
Is it possible to use more realistic tags in the stubs? Given it's used for test, not a priority fix.

Reply:
Sure